README.md

```diff
@@ -54,3 +54,23 @@ For documentation see the
 developer version or the most recent released
 <a href="https://scitools-iris.readthedocs.io/en/stable/">stable</a> version.
 </p>
+
+## [#ShowYourStripes](https://showyourstripes.info/s/globe)
+
+<h4 align="center">
+  <a href="https://showyourstripes.info/s/globe">
+    <img src="https://raw.githubusercontent.com/bjlittle/geovista-media/2023.01.1/media/readme/showyourstripes.png"
```

Comment:
When `geovista` migrates to the `SciTools` organisation, we can update the URL to this asset.

Reply:
This must be hosted somewhere already, right?! You've linked to their wonderful website - is there nothing there that we can reference directly?

Reply:
https://showyourstripes.info/stripes/GLOBE---1850-2021-MO.png ?

Reply:
It also gets updated every year

Reply:
<img src="https://showyourstripes.info/stripes/GLOBE---1850-2021-MO.png" width="3780" height="50">

Reply:
Ugh, I give up - doesn't look like there's a way of doing this in HTML without blurring.

Reply:
@trexfeathers Pushed a stable workaround... I think, after a wee bit of detective work :male_detective: 

See https://github.com/SciTools/iris/pull/5141/commits/b0e7e1d99387e67eb4ee671eb943a47c45c77d7d

Reply:
OK, if the blurring isn't a problem for you then I'm fine with it too.

(I did some reading about cropping in HTML but it needs CSS and that's not something that works in GitHub Markdown. SVG would also avoid the problem but I don't want to create more work for Prof. Hawkins!)

Reply:
SVG would be ideal, but i'm not really seeing any blurring that's obviously detrimental or a blocker